.github/workflows/Deploy

```diff
@@ -0,0 +1,65 @@
+name: Deploy
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          global-json-file: "./global.json"
+
+      - name: Restore dependencies
+        run: dotnet restore
+
+      - name: Set Version
+        run: |
+          echo "${{ github.ref }}"
+          $tagVersion = "${{ github.ref }}".substring(11)
+          echo "buildVersion=$tagVersion.${{ github.run_number }}" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
```

Comment:
This works, but there is a simpler syntax.
```suggestion
          echo "buildVersion=$tagVersion.${{ github.run_number }}" >> $env:GITHUB_ENV
```

Alternatively you can use `$env:GITHUB_OUTPUT`, give this step an id and reference the output explicitly.